scripts/assets/karma.conf.js

```diff
@@ -0,0 +1,126 @@
+// Karma configuration file, see link for more information
+// https://karma-runner.github.io/1.0/config/configuration-file.html
+
+let options = process.env.KARMA_OPTIONS;
+options = options ? JSON.parse( options ) : {};
+
+module.exports = function ( config ) {
+	const browsers = config.browsers.length === 0 ? [ 'Chrome' ] : config.browsers;
+
+	config.set( {
+		basePath: '',
+		frameworks: [ 'jasmine', '@angular-devkit/build-angular' ],
+		plugins: getPlugins(),
+		client: {
+			clearContext: false, // leave Jasmine Spec Runner output visible in browser
+			captureConsole: false,
+			jasmine: {
+				random: false
+			}
+		},
+		coverageIstanbulReporter: {
+			dir: require( 'path' ).join( __dirname, '../coverage' ),
+			reports: [ 'html', 'lcovonly' ],
+			fixWebpackSourcePaths: true
+		},
+		reporters: [ 'spec' ],
+		port: 9876,
+		colors: true,
+		logLevel: config.LOG_INFO,
+		autoWatch: true,
+		browsers,
+		singleRun: !options.watch,
+
+		concurrency: 2,
+		captureTimeout: 60000,
+		browserDisconnectTimeout: 60000,
+		browserDisconnectTolerance: 3,
+		browserNoActivityTimeout: 60000,
+
+		specReporter: {
+			suppressPassed: true,
+			suppressErrorSummary: true,
+			maxLogLines: 8
+		},
+
+		...( options.url && { files: [ options.url ] } ),
+
+		customLaunchers: {
+			BrowserStack_Edge: {
+				base: 'BrowserStack',
+				os: 'Windows',
+				os_version: '10',
+				browser: 'edge'
+			},
+			BrowserStack_IE11: {
+				base: 'BrowserStack',
+				os: 'Windows',
+				os_version: '10',
+				browser: 'ie',
+				browser_version: '11.0'
+			},
+			BrowserStack_Safari: {
+				base: 'BrowserStack',
+				os: 'OS X',
+				os_version: 'Big Sur',
+				browser: 'safari'
+			}
+		},
+
+		browserStack: {
+			username: process.env.BROWSER_STACK_USERNAME,
+			accessKey: process.env.BROWSER_STACK_ACCESS_KEY,
+			build: getBuildName(),
+			project: 'ckeditor4'
+		},
+	} );
+};
+
+// Formats name of the build for BrowserStack. It merges a repository name and current timestamp.
+// If env variable `TRAVIS_REPO_SLUG` is not available, the function returns `undefined`.
+//
+// @returns {String|undefined}
+function getBuildName() {
```

Comment:
Build name should be inferred from GHA workflow. We already set `BUILD_SLUG` env variable there. Also, we can get rid of Travis CI completely. It should be fairly simple.